src/index.js

```diff
@@ -1 +1,34 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { postScores, getScore } from './api.js';
+
+const renderPage = document.querySelector('.render-page');
+const form = document.querySelector('#form');
+const name = document.querySelector('#name');
+const number = document.querySelector('#number');
+const refreshBtn = document.querySelector('#refresh');
+
+const createScore = async () => {
+  const data = await getScore();
+  const dataInfo = data.result;
+  renderPage.innerHTML = '';
+  dataInfo.forEach((record) => {
+    renderPage.innerHTML += `<li>${record.user}: ${record.score}</li>`;
+  });
+};
```

Comment:
✅ Ensured the usage of **arrow functions** instead of the `function` keyword 👍🏻 
✅ Use `async` and `await` with the `fetch` method to send and receive data 👍🏻